install/kubernetes/helm/istio/charts/sidecar-injector/templates/configmap.yaml

```diff
@@ -16,6 +16,8 @@ data:
       - name: istio-init
         image: {{ .Values.global.hub }}/proxy_init:{{ .Values.global.tag }}
         args:
+        - "-b"
```

Comment:
what was that flag ?

Reply:
It's new to the script.  It identifies the inbound ports to be forwarded to envoy.  The wildcard (*) means everything.